dockstore-webservice/src/main/java/io/dockstore/webservice/Helper.java

```diff
@@ -131,11 +131,11 @@ public String getContent() {
         return currentList;
     }
 
-    public static List<Container> getQuayContainers(HttpClient client, Logger LOG, ObjectMapper objectMapper, List<String> namespaceList,
+    private static List<Container> getQuayContainers(HttpClient client, Logger LOG, ObjectMapper objectMapper, List<String> namespaces,
```

Comment:
Do not pass around logger objects. Just create them statically per-class and use them there.